scripts/python-skywater-pdk/skywater_pdk/generate-resources-rst.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 SkyWater PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import argparse
+from pathlib import Path
+import pandas as pd
+
+
+def parse_entries(filepath, sheet_name, cols):
+    values = pd.read_excel(
+        filepath,
+        sheet_name=sheet_name,
+        header=1,
+        usecols=cols
+    )
+    values = values.where(values.notnull(), None)
+    return values
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(prog=argv[0])
+    parser.add_argument(
+        'input_xlsx',
+        help='The path to input XLSX file or GID to Google Spreadsheet',
+        type=Path
+    )
+    parser.add_argument(
+        'output',
+        help='The path to output RST file',
+        type=Path
+    )
+    parser.add_argument(
+        '--input-is-spreadsheet-id',
+        help='The input_xlsx argument holds public Google Spreadsheet ID',
+        action='store_true'
+    )
+
+    args = parser.parse_args(argv[1:])
+
+    if args.input_is_spreadsheet_id:
+        args.input_xlsx = f'https://docs.google.com/spreadsheets/d/{args.input_xlsx}/export?format=xlsx'  # noqa: E501
+
+    news_articles = parse_entries(args.input_xlsx, 'News Articles', 'B:D')
+    talk_series = parse_entries(args.input_xlsx, 'Talk Series', 'B:M')
+    conferences = parse_entries(args.input_xlsx, 'Conferences', 'B:D')
+    linkedin = parse_entries(args.input_xlsx, 'LinkedIn Posts', 'B:C')
+    courses = parse_entries(args.input_xlsx, 'Courses', 'B:D')
+
+    with open(args.output, 'w') as out:
```

Comment:
We should probably use a [templating language (jinja preferred)](https://jinja.palletsprojects.com/en/2.11.x/) for this?

Reply:
I added usage of Jinja